tractor/_debug.py

```diff
@@ -126,7 +126,8 @@ def release(cls):
         try:
             # sometimes the ``trio`` might already be terminated in
             # which case this call will raise.
-            cls.local_pdb_complete.set()
+            if cls.local_pdb_complete is not None:
```

Comment:
This is history from https://github.com/goodboy/tractor/pull/324 which we can probably land first.